sig-multitenancy/auth/oidc-Identity-provider.md

```diff
@@ -26,7 +26,7 @@ data:
            clientID: '********'
            clientSecret: '********'
            issuer: https://accounts.google.com
-           redirectURL:  'http://ks-console/oauth/redirect'
+           redirectURL:  'http://ks-console/oauth/redirect/google'
```

Comment:
It would be great if you could provide some references here. For example, document, code line. Because I don't familiar with the feature. I cannot determine if it's correct.